### PR TITLE
Use kebab case for custom html events

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
This PR addresses some issues introduced with #931

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: frontend/src/components/icons/IconBase.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@ SPDX-License-Identifier: Apache-2.0
   <svg xmlns="http://www.w3.org/2000/svg"
     :width="width"
     :height="height"
-    :view-box="viewBox"
+    :viewBox="viewBox"
```
Comment (holgerkoser) [inline frontend/src/components/icons/IconBase.vue:11]: I think we should add a comment that this must be camelCase.

**Comment by holgerkoser** [comment]
The lifecycle hooks https://github.com/gardener/dashboard/blob/1538be54ab96c76b32f8c5ad556a195554d24056/frontend/src/mixins/asyncRef.js#L48-L49 must also be camelCase. I am not sure I you don't touched them by intention or by accident.

**Review by holgerkoser — APPROVED** [review]
/lgtm

File: frontend/src/components/icons/IconBase.vue (hunk 1 above)
Comment (grolu) [reply frontend/src/components/icons/IconBase.vue:12]: agreed & done

**Pull request merged**